tests/retry_task.go

```diff
@@ -44,12 +44,12 @@ func (t *RetryTask) GetExecution() (execute time.Time) {
 
 // GetTimeout get timeout of execution
 func (t *RetryTask) GetTimeout() (executeTimeout time.Duration) {
-	return time.Second
+	return time.Millisecond
```

Comment:
This is reduced to time.Millisecond because retry task may content with the existing lock if the lock stands for too long. For example:

Task A is a retry task and will be executed 3 times for every 1 second, each execution requires about 1 second. However, the lock of Task A is set to 1 minute.

```
    | Task A lock -----------------1 minute----------------------------->
Task A start ----->| ----> Task A retry 1  fail ---x      |
|       1 second   |          because Task A is locked.   |
```